CHANGELOG.md

```diff
@@ -1,4 +1,276 @@
+#### Models Removed
+`GenericQueryAggregation`, `ProjectCreation`, `ProjectName`, `QueryLarge`, `QueryNoticesResult`, `UpdateCollection`, `UpdateEnrichment`
+#### Model Properties Changed
+`CollectionDetails`
+* `collectionID` read only
+* `created` read only
+
+`QueryResponse`
+* `passages` added
+
+`TrainingExample`
+* `created` read only
+* `updated` read only
+
+`TraningQuery`
+* `created` read only
+* `updated` read only
+* `queryID` read only
+
+### LanguageTranslator Instance
+* No changes
+
+### LanguageTranslatorV3 Models
+
+#### Models Removed
+`TranslateRequest`
+
+### NaturalLanguageClassifierV1 Instance
+* No changes
+
+### NaturalLanguageClassifierV1 Models
+
+#### Models Removed
+`ClassifyCollectionInput`
+
+### NaturalLanguageUnderstanding Instance
+* No changes
+
+### NaturalLanguageUnderstandingV1 Models
+
+#### Models Added
+`FeaturesResultsMetadata`
+#### Models Removed
+`AnalysisResultsMetadata`, `MetadataOptions`, `Parameters`
+#### Model Properties Changed
+`AnalysisResults`
+* `metadata` type changed from `AnalysisResultsMetadata?` to `FeaturesResultsMetadata?`
+
+`Features`
+* `metadata` type changed from `MetadataOptions?` to `[String: JSON]?`
+
+### PersonalityInsights Instance
+* `profile()`: csvHeaders parameter added
+
+* `profileAsCSV()`: return type changed from `WatsonResponse<String>?` to `WatsonResponse<Data>?`
+
+### PersonalityInightsV3 Models
+* No changes
+
+### SpeechToTextV1 Instance
+* `trainLanguageModel()`: return type changed from `WatsonResponse<Void>?` to `WatsonResponse<TrainingResponse>?`
+
+* `trainAcousticModel()`: return type changed from `WatsonResponse<Void>?` to `WatsonResponse<TrainingResponse>?`
+
+### SpeechToTextV1 Models
+* No changes
+
+#### Models Removed
+`CreateAcousticModel`, `CreateLanguageModel`, `CustomWords`
+
+### TextToSpeechV1 Instance
+* `createVoiceModel()`: function changed to `createCustomModel()` now returning `WatsonResponse<CustomModel>?`
+
+* `listVoiceModels()`: function changed to `listCustomModels()` now returning `WatsonResponse<CustomModels>?`
+
+* `updateVoiceModel()`: function changed to `updateCustomModel()`
+
+* `getVoiceModel()`: function changed to `getCustomModel()` now returning `WatsonResponse<CustomModel>?`
+
+* `deleteVoiceModel()`: function changed to `deleteCustomModel()`
+
+### TextToSpeechV1 Models
+
+#### Models Added
+`CustomModel`, `CustomModels`
+#### Models Removed
+`CreateVoiceModel`, `Text`, `UpdateVoiceModel`, `VoiceModel`, `VoiceModels`
+#### Model Properties Changed
+`Voice`
+* customization type changed from `VoiceModel?` to `CustomModel?`
+
+### ToneAnalyzer Instance
+* `tone()`: `tones` parameter added
+
+### ToneAnalyzerV3 Models
+
+#### Models Removed
+`ToneChatInput`
+
+### VisualRecognitionV3 Instance
+* No changes
+
+### VisualRecognitionV3 Models
+* No changes
+
+### VisualRecognitionV4 Instance
+* `createCollection()`: `trainingStatus` parameter added
+
+* `updateCollection()`: `trainingStatus` parameter added
```

Comment:
Training status should be removed from createCollection and updateCollection - these should be readonly. need to address this in the generator but please remove for now

Reply:
My had removed them in node but forgot about them here